### ISSUE TITLE
confusing intro text

### ISSUE THREAD

**Issue by danielskatz** [jupyter/nbclient#84]
The text on https://nbclient.readthedocs.io/en/latest: "Similar in nature to jupyter_client, as the jupyter_client is to the jupyter protocol nbclient is to notebooks allowing for execution contexts to be run." is fairly hard to parse. I was going to put in a PR to make it more clear, but I couldn't quite parse it well enough to figure out how to do so.

**Comment by choldgraf** [comment]
Yeah that's a good point - I kind of feel like we should move the `Similar in nature` stuff to a deeper page, as most users won't know what `jupyter_client` or the `jupyter protocol` are. We should be able to explain nbclient in language that is intuitive to someone with a very high-level understanding of Jupyter
